packages/ckeditor5-media-embed/tests/mediaembedui.js

```diff
@@ -142,7 +149,7 @@ describe( 'MediaEmbedUI', () => {
 				sinon.assert.calledOnce( spy );
 			} );
 
-			it( 'executes the command and closes the UI (if the form is valid)', () => {
+			it( 'executes the command and closes the UI (if the form is valid)', done => {
```

Comment:
Tests for media embed have the same problem when running without focus as the ones fixed in bacb8cb.

Reply:
Addressed in aa85b9f.